### DISCUSSION TITLE
how to specify font size relative to staff size?

### DISCUSSION THREAD

**Discussion by lemzwerg** [w3c-cg/musicxml#524] [Q&A] [answered]
AFAICS, the `font-size` attribute always specifies the absolute size of an object, that is, the given font size doesn't change if a score's `<scaling>` value (in `<defaults>`) or a staff's `<staff-size>` (in `<staff-detail>`) is changed later on (BTW, it would be nice if the documentation explicitly mentioned that).  Even if CSS values like `x-large` are used they seem to be application-specific but still absolute.

Is there a way to specify font size values relative to the size of a staff so that they get automatically rescaled if someone adjusts the staff or score scaling?

**Comment by mdgood** [comment, marked as answer]
Alas, there isn't. This is something that I believe will be fixed in MNX, as it is more important for MNX's use cases.

**Reply by lemzwerg** [reply]
Thanks.
